### ISSUE TITLE
Computing classification probabilities

### ISSUE THREAD

**Issue by ghost** [nitishsrivastava/deepnet#70]
How to compute classification probabilities using deepnet?

**Comment by jormansa** [comment]
You can use the extract_neural_net_representation.py program to get the values on the last layer (softmax).

**Comment by ghost** [comment]
@jormansa Thanks. I'm using dbm to build a model. Can I use extract_dbn_representation.py to get each layer representation of the input? Though this script gets the name dbn, it seems that it also uses the mean-field and variational method (positive phrase) to get the posterior probabilities. I think in dbn people usually get each layer representation as in neural net, unlike what happens here. How it goes?

**Comment by jormansa** [comment]
Sorry, I don't know how to do that with a DBM. Maybe you can create your own program inspired by extract_dbn_representation.py
